aws-android-sdk-iot/pom.xml

```diff
@@ -44,6 +44,11 @@
       <version>1.10.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.conscrypt</groupId>
+      <artifactId>conscrypt-android</artifactId>
+      <version>2.0.0</version>
```

Comment:
```
<type>aar</type>
```

Because of all the integration failures that we have seen this change cause in integration tests and  unit tests, has the maven build of this library been tested in an app?

Reply:
It has been tested.